output.proto

```diff
@@ -64,8 +64,13 @@ message SystemHealth {
       SENSOR_TILTED = 3;
       SENSOR_SUSPENDED = 4;
     }
+    message SensorInfo {
+      SensorStatus status = 1;
+      string topic = 2;
+      string sensor_name = 3;
+    }
     Status status = 1;
-    map<string, SensorStatus> sensors = 3;
+    map<string, SensorInfo> sensors = 3;
```

Comment:
Just to be safe, let's reserve 3 and change the ID to 4. Reusing the id is strictly forbidden. Then this would not break the existing client applications.